src/main/java/com/clevertap/apns/Notification.java

```diff
@@ -45,20 +45,29 @@
     private final String token;
     private final String topic;
     private final String collapseId;
+    private final Long expiration;
+    private final Integer priority;
```

Comment:
Let's make this into an enum? The values could be Immediate, and PowerConsideration